tests/test_base.py

```diff
@@ -3,9 +3,20 @@
 import aiohttp
 import pytest
 
-from idna_ssl import patch_match_hostname, reset_match_hostname
+from idna_ssl import PY_370, patch_match_hostname, reset_match_hostname
 
+skip_370 = pytest.mark.skipif(
+    PY_370,
+    reason='Python>=3.7.0 do not need monkey patching',
+)
 
+only_370 = pytest.mark.skipif(
+    not PY_370,
+    reason='Python>=3.7.0 do not need monkey patching',
```

Comment:
This message is exactly the same as one on line 10. One of them is clearly wrong.